openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java

```diff
@@ -300,9 +300,9 @@ private Schema mapEnumValues(EnumSymbol enumSymbol) {
         List<ConstantSymbol> enumMembers = enumSymbol.members();
         for (ConstantSymbol enumMember : enumMembers) {
             if (enumMember.typeDescriptor().typeKind() == TypeDescKind.SINGLETON) {
-                enums.add(enumMember.typeDescriptor().signature());
+                enums.add(enumMember.typeDescriptor().signature().replaceAll("\"", ""));
```

Comment:
But this is a breaking change, right? I think we need to check the reason for this change first and fix here only if intentional (e.g., a bug fix). Could be a semantic API change or a lang change.

cc @gimantha @dulajdilshan

Reply:
@MaryamZi yeah , when I'm using `2201.0.3-rc1.4` , it returns `enumMember.typeDescriptor().signature()` , `PATCH`. 
But with new version it returns  `"PATCH"`

Reply:
Here the behaviour of the Semantic API is correct. The signature of the string literals must contain `"`. I think adding this change on Open API side  is okay since this doesn't interfere with the Semantic API behaviour

Reply:
The reason for this change is that previously we missed `"` in string literals. We added this with this PR https://github.com/ballerina-platform/ballerina-lang/pull/35640

Reply:
But this is the signature, right? Not the value?

If we had 

```ballerina
const FOO = "foo";
```

if we get the signature of type `FOO`, wouldn't/shouldn't it be `FOO`?

In https://github.com/ballerina-platform/ballerina-lang/issues/35407, there was no type definition/constant declaration for `"parent"`, so there we have to use `"parent"`. This is different, right?

Reply:
Yeah. Its `"FOO". Here, shouldn't the typedef of the enum contain `"` since it is a string value?

Reply:
```ballerina
enum Status {
    OPEN,
    CLOSED
}
```

is the same as

```ballerina
const OPEN = "OPEN";
const CLOSED = "CLOSED";

type Status OPEN|CLOSED;
```

When we do `io.ballerina.compiler.api.symbols.EnumSymbol#members`, shouldn't the behaviour be the same as that of union member types?

Reply:
@lnash94 Can you try using `enumMember.get(0).signature()` this instead of  `enumMember.typeDescriptor().signature()`

Reply:
Tried with `enumMember.signature()`,  it will return Enum constant instead of its value,  when we have an enum with a value like below way,
Enum with value:
```ballerina
enum Action {
    GET = "get" ,
    POST = "post",
    PUT = "put",
    DELETE = "delete",
    PATCH = "patch"
}
```
expected:
```openapi
enum:
            - patch
            - delete
            - put
            - post
            - get
```
Recived :
```openapi
enum:
          - PATCH
          - DELETE
          - PUT
          - POST
          - GET
```